app/script/media/MediaEmbeds.js

```diff
@@ -42,18 +43,25 @@ z.media.MediaEmbeds = (function() {
     };
 
     options = _.extend(defaults, options);
-    const iframe_container = `<div class="{0}"><iframe class="${options.type}" width="{1}" height="{2}" src="{3}" frameborder="{4}"{5}></iframe></div>`;
 
     if (!options.video) {
-      options.allowfullscreen = '';
+      options.allowfullscreen = false;
       options.class = 'iframe-container';
     }
 
-    if (z.util.Environment.desktop) {
-      options.allowfullscreen = '';
-    }
+    return z.util.StringUtil.format((() => {
+
+      // Build the iframe/webview
+      let iframe_container = '<div class="{0}">';
+      if(z.util.Environment.electron) {
```

Comment:
Use `z.util.Environment.desktop` instead as electron will also return true in rambox for example that does not have your implementation. We also need to figure out a way to have this work with older versions of the desktop wrapper. This will break compat with all currently released clients.